### PR TITLE
Update laplace_transform_839973EF5D.tex

### PR DESCRIPTION
**Pull request by fs446**
general Ansatz for impulse response via inverse laplace in sec. 1.5.1

### CODE REVIEW

File: tutorial_extended_latex_deu/laplace_transform_839973EF5D.tex [hunk 1]
```diff
@@ -1369,11 +1369,105 @@ \subsubsection{Task a) with Laplace Transform / Impulse Response}
...
+Splitting into real and imaginary part we can rewrite
+\begin{align}
+h(t) = \frac{K}{\Im\{s_{\infty,1}\}} \cdot \e^{\Re\{s_{\infty,1}\} t} \cdot \frac{\e^{\im \Im\{s_{\infty,1}\} t}   - \e^{-\im \Im\{s_{\infty,1}\} t}}{2\im} \cdot \epsilon(t)
+=
+\frac{K}{\Im\{s_{\infty,1}\}} \cdot \e^{\Re\{s_{\infty,1}\} t} \cdot \sin(\Im\{s_{\infty,1}\} t) \cdot \epsilon(t)
+\end{align}
+%
+This nice general notation allows for convenient interpretation.
+%
+The gain factor of the system $K$ is scaled by the angular frequency of the sine, which
```
Comment (robhau) [inline tutorial_extended_latex_deu/laplace_transform_839973EF5D.tex]: Maybe "is scaled by the inverse angular frequency" is better?

File: tutorial_extended_latex_deu/laplace_transform_839973EF5D.tex [hunk 2]
```diff
@@ -1369,11 +1369,105 @@ \subsubsection{Task a) with Laplace Transform / Impulse Response}
...
+is encoded in $\Im\{s_{\infty,1}\}$.
+%
+The poles should be in the left $s$-plane, hence $\Re\{s_{\infty,1}\}<0$, such that
+the exponential function is decaying and not uprising.
```
Comment (robhau) [inline tutorial_extended_latex_deu/laplace_transform_839973EF5D.tex]: I think this sentence implies that the exponential function is uprising for Re{s}>=0, but for Re(s)=0 it is neither uprising nor decaying.

File: tutorial_extended_latex_deu/laplace_transform_839973EF5D.tex (hunk 1 above)
Comment (fs446) [reply tutorial_extended_latex_deu/laplace_transform_839973EF5D.tex]: good point, exactly taken the suggestion

File: tutorial_extended_latex_deu/laplace_transform_839973EF5D.tex (hunk 2 above)
Comment (fs446) [reply tutorial_extended_latex_deu/laplace_transform_839973EF5D.tex]: good point, changed to "such that the exponential function is decaying.", which is what we want to have in practice.

**Pull request merged**